### PR TITLE
New Colors, Math Stuff and More Docs [Fixed]

### PR DESCRIPTION
**Pull request by nmsderp**
- Added new color ,"Crimson."
- Added math stuff (value of pi built in)
- Docs for the math stuff (and updated color docs).

### CODE REVIEW

**Review by dimkauzh — COMMENTED** [review]
Hey, so I will merge this code, but I want to tell you something: you cannot use - in python, because python thinks that's a minus, you can only use _. For now it's no prob I will fix it but please next time check by yourself if your code is right. But thanks for the contribution!

File: src/fusionengine/files/fusion-math.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# this is used for extra debugging features
+# tbh this might get removed
+class Fusion-math:
+    # most likely useless but i put it here anyway
+    self.VALUE-OF-PI = 3.141592653589793238462643383279502884197.
```
Comment (onepersonintheuniverse) [inline src/fusionengine/files/fusion-math.py]: # ***__`what`__***

Comment (onepersonintheuniverse) [reply src/fusionengine/files/fusion-math.py]: how did this get merged

**Review by onepersonintheuniverse — COMMENTED** [review]
you can't hyphen a variable in Python 3, all hail the `_`

File: src/fusionengine/files/fusion-math.py (hunk 1 above)
Comment (dimkauzh) [reply src/fusionengine/files/fusion-math.py]: I fixed it when I was merging it lol

**Pull request merged**